File: Java/453. Minimum Moves to Equal Array Elements.java
class Solution {
    public int minMoves(int[] nums) {
        int res = 0;
		int m = Integer.MAX_VALUE;
		for(int i = 0; i<nums.length; i++){
            m = Math.min(m,nums[i]);
        }
		for(int i = 0; i<nums.length; i++){
            res+=nums[i]-m;
        }
		return res;
    }
}

File: C#/453. Minimum Moves to Equal Array Elements.cs
public class Solution {
    public int MinMoves(int[] nums) {
        int res = 0;
		int m = Int32.MaxValue;
		for(int i = 0; i<nums.Length; i++){
            m = Math.Min(m,nums[i]);
        }
		for(int i = 0; i<nums.Length; i++){
            res+=nums[i]-m;
        }
		return res;
    }
}
